wheels/tests/view/text/excerpt.cfc

```diff
@@ -26,7 +26,7 @@ component extends="wheels.tests.Test" {
 		args.phrase = "excerpt view helper";
 		args.radius = "10";
 		e = _controller.excerpt(argumentcollection = args);
-		r = "[more]sting the excerpt view helper to see if[more]";
+		r = "[more]testing the excerpt view helper to see if[more]";
```

Comment:
@MvdO79 can you revert this one. This is causing this test to fail which is failing the PR.